src/Castle.Facilities.Logging/LoggingFacility.cs

```diff
@@ -165,21 +170,25 @@ public LoggingFacility ToLog(string name)
 		}
 
 #if CASTLE_SERVICES_LOGGING
+		[Obsolete("A logger factory implementation type should be provided via LogUsing<T>(), this will be removed in the future.")]
 		public LoggingFacility UseLog4Net()
 		{
 			return LogUsing(LoggerImplementation.Log4net);
 		}
 
+		[Obsolete("A logger factory implementation type should be provided via LogUsing<T>(), this will be removed in the future.")]
```

Comment:
There are currently 131 Obsolete attributes in the code base. Can't we just delete this altogether? Assuming we kick the version up to v5.